blaze/expr/tests/test_arithmetic.py

```diff
@@ -64,8 +64,8 @@ def test_dir():
     i = symbol('i', '10 * int')
     d = symbol('d', '10 * datetime')
 
-    assert i + 1
-    with pytest.raises(Exception):
+    i + 1  # this works
```

Comment:
Nothing in particular, what was wrong with what was there? Bare code that isn't assigned to anything or used anywhere increases the likelihood that I will delete it and that could potentially hide regressions. Likely a personal shortcoming of mine, but I think it's easy just to keep the assert